ci/show-python-package-versions.py

```diff
@@ -2,7 +2,7 @@
 import sys
 
 DEPENDENCIES = ['numpy', 'scipy', 'sklearn', 'matplotlib', 'nibabel',
-                'ply', 'pandas']
+                'ply', 'pandas', 'biopython']
```

Comment:
will drop. will add try/except import in the Dataset and keep it in the Docker image.